<launch>
  <arg name="debug" default="false" />
  
  <arg name="launch_prefix" value="xterm -e gdb --args" if="$(arg debug)" />
  <arg name="launch_prefix" value="" unless="$(arg debug)" />
  
  <node launch-prefix="$(arg launch_prefix)" pkg="dvo_ros" type="camera_tracker" name="camera_tracker" output="screen">
    <remap from="camera/rgb/image_rect" to="camera/rgb/image_rect_color" />
    <remap from="camera/rgb/camera_info" to="camera/rgb/camera_info"/>
    <remap from="camera/depth_registered/image_rect_raw" to="camera/depth/image_rect" />
    <remap from="camera/depth_registered/camera_info" to="camera/depth/camera_info"/>

    <param name="reconstruction" type="bool" value="true" />
    <param name="use_weighting" type="bool" value="true" />
  </node>
  
  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" args="/camera_tracker" />
  <node name="rviz" pkg="rviz" type="rviz" />
</launch>
